mysql-test/suite/rocksdb.sys_vars/t/rocksdb_write_batch_max_bytes_basic.test

```diff
@@ -0,0 +1,26 @@
+--source include/have_rocksdb.inc
```

Comment:
Ideally should go to the rocksdb main suite, and sys_vars should have the usual red tape test of session/global, ro/rw, type, range testing. I'm not asking you to make this change unless you feel this is necessary.

Reply:
Yeah, I kind of thought that too when I was touching up the test on import but left as-is for future merge. Like other tests, probably not a bad idea to create a JIRA issue to suggest and track the change/split, then send it upstream.

Reply:
Created https://jira.percona.com/browse/MYR-189